### DISCUSSION TITLE
How To Target repo to .net 6 only

### DISCUSSION THREAD

**Discussion by RealityMasque** [Megabit/Blazorise#5061] [Q&A] [answered]
Hi,

I downloaded the repo as a zip so I can see how to have a checkbox column that has a select all checkbox in the header.  Unfortunately, we only have .net 6 on our machines.  I've been trying to change all projects referenced by the Blazorise.Demo project to .net 6.  I have also unloaded projects not referenced by the Blazorise.Demo project.  But I still get build errors.  IS there an easier way to make the solution, or Blazorise.Demo project, build for .net 6 only?

Thanks,

Orion

**Comment by David-Moreira** [comment, marked as answer]
Hello @RealityMasque 
Why don't you try our templates?

This will create a project in whatever dotnet version you want (from the supported ones) that should be ready to go.

https://blazorise.com/docs/templates

**Comment by stsrki** [comment]
Check the `Blazorise.props` file under the `/Build` folder. There are defined all the framework versions.
